### ISSUE TITLE
[Feature] Detachable videos

### ISSUE THREAD

**Issue by HYPExMon5ter** [JsSucks/BetterDiscordApp#323]
- [x] Used appropriate template for the issue type
- [x] Searched both open and closed issues for duplicates of this issue
- [x] Title adequately and _concisely_ reflects the feature or the bug

**Is your feature request related to a problem? Please describe.**

No

**Describe the solution you'd like**

A button to detach a video so you can watch the youtube video/gif/any other video files your friend sent you while replying to someone else without it stopping. There were a couple plugins like this in BDV1 but they're all outdated and not working, and I feel as this should be in core BD anyways.

**Describe alternatives you've considered**

**Additional context**

**Are you willing and able to implement this?**

If I knew how, I would.

**Comment by intrnl** [comment]
This is better off done by a plugin, not by BetterDiscord itself.

**Comment by HYPExMon5ter** [comment]
Why's that? This feature would get used by everyone atleast once lol

**Comment by zerebos** [comment]
I agree wtih @intrnl. I can say with certainty that I personally wouldn't use that feature. Can you give a reason why that should be part of the core as opposed to a plugin?

**Comment by HYPExMon5ter** [comment]
I just feel most users would benefit from this feature 😛 but if you think this should be a plugin instead I understand, it’s just finding someone willing to make it.

**Comment by snappyapple632** [comment]
Key word here: most. Most but not all users would see a benefit, and therefore it should be provided as an option as a plugin.

The same could be said about emotes, E2EE, etc but they are also optional to an extent. I can see E2EE, for example, being better as a core feature since it's about security and is more refined versus the DiscordCrypt plugin for v1.

After all, BD is about enhancing the Discord experience by letting the user **choose** what they want enhanced. If people want videos away from the Discord window, go ahead and make a plugin for it!

**Comment by HYPExMon5ter** [comment]
That’s the thing, I would make a plugin for it if I knew how. I don’t have the time to learn how currently though.
